explanations/machine-learning-terms.md

```diff
@@ -41,6 +41,14 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>Technically, it's a statistical procedure that identifies and sorts a small number of uncorrelated variables, called principal components, in such a way that the first one has the most scattered data. So if we want to reduce from <tt>n</tt> to <tt>k</tt> dimensions, we keep the <tt>k</tt>'s first principal components. But the lower is <tt>k</tt> the greater is the information loss.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Kernel</td>
```

Comment:
I would reformulate the title as "Kernel (function)". This is because there are different meanings of kernels in machine learning depending on the context. The other one I wanted to write about is a kernel matrix which is typical for convolutional neural networks.